### PR TITLE
src: sink: Implement SDP generation for sink interface

### LINKED ISSUE #17: Create sdp files for stream (Open stream on VLC)
This helps vlc and others programs to run streams without much user configuration

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
* UDP - works with VLC and GStreamer for H264, RAW, and MJPG.
* RTPS - both VLC and GStreamer parse it, but they refuse to play it.

closes #17

### CODE REVIEW

**Review by patrickelectric — COMMENTED** [review]
Can we add an endpoint that provide the sdp files ?

**Comment by joaoantoniocardoso** [comment]
image

File: src/stream/sink/udp_sink.rs [hunk 1]
```diff
@@ -142,7 +182,7 @@ impl UdpSink {
             })
             .collect::<Vec<String>>()
             .join(",");
-        let description = format!("multiudpsink clients={addresses}");
```
Comment (patrickelectric) [inline src/stream/sink/udp_sink.rs:145]: try to avoid creating unnecessary noise in commits that are unrelated to the functional change.
you can do it in individual commits.

**Review by patrickelectric — APPROVED** [review]

File: src/stream/sink/udp_sink.rs (hunk 1 above)
Comment (joaoantoniocardoso) [reply src/stream/sink/udp_sink.rs:145]: This is related: because the property self.addresses was added, the meaning of `addresses` would be confusing in the context of this function.

Comment (patrickelectric) [reply src/stream/sink/udp_sink.rs:145]: Oh, I see it now, thanks!
